tests/test_models.py

```diff
@@ -30,9 +30,24 @@ def test_tile_matrix_set(tileset):
         ts = TileMatrixSet.model_validate_json(f.read())
     # This would fail if `crs` isn't supported by PROJ
     assert isinstance(ts.crs._pyproj_crs, pyproj.CRS)
+    assert isinstance(ts.geographic_crs, pyproj.CRS)
     assert repr(ts)
 
 
+@pytest.mark.parametrize("tileset", tilesets)
+def test_geographic_crs_bbox(tileset):
+    """check that geographic bounds are correct."""
+    with open(tileset, "r") as f:
+        ts = TileMatrixSet.model_validate_json(f.read())
+
+    if not pyproj.CRS.from_epsg(4326) == ts.geographic_crs:
```

Comment:
for some TMS, the `geographic_crs` returned by `tms._pyproj_crs.geodetic_crs` isn't strictly equal to `WGS84`, but by checking the `geographic transformation` of the tms bbox we can see they give the same result